sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_type.py

```diff
@@ -0,0 +1,58 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import pytest
+from azure.ai.formrecognizer._helpers import get_content_type
+from testcase import FormRecognizerTest
+
+
+class TestContentType(FormRecognizerTest):
+
+    def test_pdf(self):
```

Comment:
Can you add a test that ensures that passes a content type (wusing enum), and make sure it overwrites the suggest content type?

Reply:
done